res/values/colors.xml

```diff
@@ -94,7 +94,6 @@
 
     <!-- Background Color of the selected week for the mini calendar on the tablet size -->
     <color name="month_selected_week_bgcolor">#FFDDDDDD</color>
-    <color name="month_today_bgcolor">#25a2ff00</color>
```

Comment:
Isn't that still used here? https://github.com/Gitsaibot/Etar-Calendar/blob/MonthTodayColor/src/com/android/calendar/month/MonthWeekEventsView.java#L246
Also there is month_today_bgcolor_black and month_today_bgcolor_dark..

Reply:
Would it make sense to do the same modification for those two themes as well?